include/RAJA/index/ListSegment.hpp

```diff
@@ -38,7 +38,7 @@
 #include "RAJA/util/macros.hpp"
 #include "RAJA/util/types.hpp"
 
-#if defined(RAJA_ENABLE_CUDA)
+#if defined(__NVCC__) && defined(RAJA_ENABLE_CUDA)
```

Comment:
Probably want `__CUDACC__` rather than `__NVCC__`

Reply:
Ah even better! Thanks.

Reply:
Your solution looks good too.

Reply:
Not sure why, but the  ``#if defined(__CUDACC__) && defined(RAJA_ENABLE_CUDA)``  combination didn't work. I reverted to the previous commit.

Reply:
Works for me.